src/evaluation/context.rs

```diff
@@ -13,22 +11,27 @@ use crate::EvaluationContextFieldValue;
 /// define rules that return a specific value based on the user's email address, locale, or the
 /// time of day. The context provides this information. The context can be optionally provided at
 /// evaluation, and mutated in before hooks.
-#[derive(Clone, TypedBuilder, Default, PartialEq, Debug)]
+#[derive(Clone, Default, PartialEq, Debug)]
 pub struct EvaluationContext {
     /// The targeting key uniquely identifies the subject (end-user, or client service) of a flag
     /// evaluation. Providers may require this field for fractional flag evaluation, rules, or
     /// overrides targeting specific users. Such providers may behave unpredictably if a targeting
     /// key is not specified at flag resolution.
-    #[builder(default, setter(into, strip_option))]
     pub targeting_key: Option<String>,
 
     /// The evaluation context MUST support the inclusion of custom fields, having keys of type
     /// string, and values of type boolean | string | number | datetime | structure.
-    #[builder(default)]
     pub custom_fields: HashMap<String, EvaluationContextFieldValue>,
 }
 
 impl EvaluationContext {
+    /// Create a new [`EvaluationContext`] with given targeting key.
+    #[must_use]
+    pub fn with_targeting_key(mut self, targeting_key: impl Into<String>) -> Self {
```

Comment:
neat!

Reply:
Thank you. I removed some (ab)use of typed builder for simple scenarios like this...